plugins/inputs/snmp/snmp_test.go

```diff
@@ -139,71 +144,76 @@ func TestFieldInit(t *testing.T) {
 }
 
 func TestTableInit(t *testing.T) {
+	testDataPath, err := filepath.Abs("./testdata")
+	require.NoError(t, err)
+
 	tbl := Table{
 		Oid: ".1.0.0.0",
 		Fields: []Field{
-			{Oid: ".999", Name: "foo"},
 			{Oid: "TEST::description", Name: "description", IsTag: true},
 		},
 	}
-	err := tbl.Init()
+	s := &Snmp{
+		ClientConfig: snmp.ClientConfig{
+			Path: []string{testDataPath},
+		},
+	}
+	err = tbl.Init(s)
 	require.NoError(t, err)
 
 	assert.Equal(t, "testTable", tbl.Name)
 
-	assert.Len(t, tbl.Fields, 5)
-	assert.Contains(t, tbl.Fields, Field{Oid: ".999", Name: "foo", initialized: true})
-	assert.Contains(t, tbl.Fields, Field{Oid: ".1.0.0.0.1.1", Name: "server", IsTag: true, initialized: true})
-	assert.Contains(t, tbl.Fields, Field{Oid: ".1.0.0.0.1.2", Name: "connections", initialized: true})
-	assert.Contains(t, tbl.Fields, Field{Oid: ".1.0.0.0.1.3", Name: "latency", initialized: true})
-	assert.Contains(t, tbl.Fields, Field{Oid: ".1.0.0.0.1.4", Name: "description", IsTag: true, initialized: true})
+	assert.Len(t, tbl.Fields, 1)
+	assert.Contains(t, tbl.Fields, Field{Oid: ".1.0.0.0.1.4", Name: "description", IsTag: true, initialized: true, snmp: s})
```

Comment:
Do not change test, all table fields should be added when only giving table OID.